LayoutTests/platform/mac-wk1/TestExpectations

```diff
@@ -1086,6 +1086,8 @@ imported/w3c/web-platform-tests/editing/run/multitest.html?5001-6000 [ Slow ]
 imported/w3c/web-platform-tests/editing/run/multitest.html?6001-7000 [ Slow ]
 imported/w3c/web-platform-tests/editing/run/multitest.html?7001-8000 [ Slow ]
 
+imported/w3c/web-platform-tests/uievents/mouse/mousemove_prevent_default_action.tentative.html [ Failure ]
```

Comment:
We get the desired `selectionchange`/`dragstart` events here, but some of the event ordering is messed up. That is a test issue (and a WPT change proposal has been filed for this)